.github/workflows/reusable-docker-arm-build.yaml

```diff
@@ -105,6 +106,21 @@ jobs:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Set local mirror
+        if: ${{ inputs.worker == 'ARM64' }}
```

Comment:
Do we need a condition?

Reply:
if we ever change workers to use like ubuntu-latest or something due to some issues with the current worker, this migth block us and we may forget it no? 

I just thougth that as in the past we had to quickly change the workers due to issues, we migth have to again in the future. I can drop it if you want?

Reply:
Cant' we have multiple labels on self hosted workers? E.g. add `self-hosted` to the arm64 ones so that we can select all self-hosted workers when needed. Because we could as well switch to non arm self-hosted runners in which case we would still want the mirror to work. Anyway, it's minor. We can improve it later.

Reply:
I think they have a self-hosted label as well, so indeed we could and then this would not apply anymore I guess?